src/hotspot/share/interpreter/bytecodes.cpp

```diff
@@ -29,13 +29,309 @@
 #include "utilities/align.hpp"
 #include "utilities/bytes.hpp"
 
-bool            Bytecodes::_is_initialized = false;
-const char*     Bytecodes::_name          [Bytecodes::number_of_codes];
-BasicType       Bytecodes::_result_type   [Bytecodes::number_of_codes];
-s_char          Bytecodes::_depth         [Bytecodes::number_of_codes];
-u_char          Bytecodes::_lengths       [Bytecodes::number_of_codes];
-Bytecodes::Code Bytecodes::_java_code     [Bytecodes::number_of_codes];
-unsigned short  Bytecodes::_flags         [(1<<BitsPerByte)*2];
+#define JVM_BYTECODES(XX)                                                                                                        \
```

Comment:
The convention for X macros in the code is usually naming the whole macro {JVM_}BYTECODES_DO and using a lower case argument.  "def" would work and look more consistent than XX.

Reply:
Done.